lib/mux/video/assets.ex

```diff
@@ -121,6 +121,57 @@ defmodule Mux.Video.Assets do
 
   """
   def update_master_access(client, asset_id, params) do
-    Base.put(client, @path <> "/assets/" <> asset_id <> "/master-access", params)
+    Base.put(client, "#{@path}/#{asset_id}/master-access", params)
+  end
+
+  @doc """
+  Create a new playback ID.
+
+  Returns `{:ok, playback_id, %Telsa.Env{}}`.
+
+  ## Examples
+
+      iex> client = Mux.Base.new("my_token_id", "my_token_secret")
+      iex> {:ok, playback_id, _env} = Mux.Video.Assets.create_playback_id(client, "00ecNLnqiG8v00TLqqeZ00uCE5wCAaO3kKc", %{policy: "public"})
+      iex> playback_id
+      #{inspect(Fixtures.playback_id)}
+
+  """
+  def create_playback_id(client, asset_id, params) do
+    Base.post(client, "#{@path}/#{asset_id}/playback-ids", params)
+  end
+
+  @doc """
+  Retrieve a playback ID.
+
+  Returns `{:ok, playback_id, %Telsa.Env{}}`.
+
+  ## Examples
+
+      iex> client = Mux.Base.new("my_token_id", "my_token_secret")
+      iex> {:ok, playback_id, _env} = Mux.Video.Assets.get_playback_id(client, "00ecNLnqiG8v00TLqqeZ00uCE5wCAaO3kKc", "FRDDXsjcNgD013rx1M4CDunZ86xkq8A02hfF3b6XAa7iE")
```

Comment:
So this line is failing because there is no mocked request for this call. See the comment below in the mocks around missing a leading `/`